pkg/scheduler/scheduler_test.go

```diff
@@ -738,7 +747,14 @@ func Test_buildQueueingHintMap(t *testing.T) {
 				return exts[i].Name() < exts[j].Name()
 			})
 
-			got := buildQueueingHintMap(exts)
+			got, err := buildQueueingHintMap(ctx, exts)
+			if err != nil {
+				if tt.wantErr != nil {
+					cmp.Diff(tt.wantErr, err.Error())
+				} else {
+					t.Fatal(err)
+				}
```

Comment:
`cmp.Diff` doesn't do anything actually.

```suggestion
				if tt.wantErr != nil && tt.wantErr.Error() != err.Error() {
					t.Fatalf("unexpected error from buildQueueingHintMap: expected: %s, actual: %s, tt.wantErr, err)
				} 
				
				if  tt.wantErr == nil {
					t.Fatal("unexpected error from buildQueueingHintMap: %s, err)
				}
```

Reply:
@googs1025 you missed this.

Reply:
fixed